### DISCUSSION TITLE
How can I remove the main image and other product images once posted?

### DISCUSSION THREAD

**Discussion by kinjalAP** [amzn/selling-partner-api-models#3400] [General]
How can I remove the main image and other product images once posted? I am using Feed Type =" POST_FLAT_FILE_LISTINGS_DATA" and trying to post and revise the product. when I pass blank value in "main_image_url" and "other_image_url1 to 8". It does not remove existing images in the product. Please help me by using which API or feed_type, I can remove existing images.

**Comment by Javadebi** [comment]
What do you pass for update_delete?
if you pass UPDATE, it will replace it

**Comment by kinjalAP** [comment]
Yes, I will pass "Update" in "update_delete". But if first time we will
pass the value in main_img and second time if we pass blank in main_img
then it will not remove main_img.

**Reply by Javadebi** [reply]
Could you share your template without your sku inside it?

**Comment by kinjalAP** [comment]
Sorry, I can't. I recently check this case, At the first time I
passed the image URL in "other_image_url1". and the image was posted
successfully, after that I again update the product with a blank value in
"other_image_url1". and also passed the "Update" value in "update_delete".
but the old posted image of "other_image_url1" remains as it is. it did not
remove it.

If I pass a different value in other_image_url1 then it will replace
with the old one but if I did not pass any value in it then it will not
remove the image.

**Reply by Javadebi** [reply]
did you use the same file you downloaded from seller central?

**Reply by wszwc** [reply]
Hello, may I ask what json data format should be used to upload a product through the feedtype "POST_FLAT_FILE_LISTINGS_DATA"? Officially only feedtypevalue

**Comment by kinjalAP** [comment]
Yes
